.gitignore

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
```suggestion
# Windows' auto-generated config files
```

Reply:
Inform users about the file type

Reply:
In C&C on the work server? 
Ok -- but where do you want me to do that?